lib/utils.js

```diff
@@ -94,6 +96,21 @@ exports.utils = {
         emit(exports, 'intent', 'destroy');
     },
     /**
+    * Calculates the amount of time remaining until the next
+    * notification timer
+    * @returns The next CTA complete time
+    */
+    getNextTimerCompleteTime: function() {
+        let lastCTATime = storageManager.get('lastSidebarCTACompleteTime');
```

Comment:
That would indeed make it easier to test. I reckon I am going to leave that as the first task for the first follow up pull request. Want to get this fixed so the folks who are manually testing can move forward. Feels crappy to put the tests off but, this would be the last time.